### DISCUSSION TITLE
Is it possible to create connected messages?

### DISCUSSION THREAD

**Discussion by dude-guy-boy** [skyra-project/discord-components#358] [Q&A]
Hi, I'm using the framework-less html version and I'm wondering if its possible to create connected messages such as how two messages sent by the same person closely together in time in discord will not show the users name and profile picture like this:
<img width="264" alt="Screenshot 2023-07-12 at 11 39 47 pm" src="https://github.com/skyra-project/discord-components/assets/87508427/ec334b9e-34df-45fe-b486-9d95a2599e89">
<img width="275" alt="Screenshot 2023-07-12 at 11 40 32 pm" src="https://github.com/skyra-project/discord-components/assets/87508427/da57351a-8980-4cf0-a3a5-4d471061c4ab">

**Comment by favna** [comment]
It is not, the best you can do is simply separate them by `<br>` elements.
